src/lib/grammar.js

```diff
@@ -129,12 +129,13 @@ function buildRegex(map, keyEx, valEx) {
         '(?<' + keyEx + '>',
             keys,
         ')',
-        // value is optional
         '(?:\\(',
             '(?<' + valEx + '>',
                 GRAMMAR.VALUES,
             ')',
-        '\\))?',
+        '\\))',
+        // value is optional only for helper
+        keyEx === 'helper' ? '?' : ''
```

Comment:
:+1: it makes sense, we can use a boolean flag instead of a type

```js
[
    {
       // no param flag is required
        "name": "Border",
        "matcher": "Bd",
        "styles": {
            "border-width": "1px",
            "border-style": "solid"
        }
    },

    {
        "name": "Line clamp",
        "hasOptionalParams": true,
        "matcher": "LineClamp",
        "styles": {
            "-webkit-line-clamp": "$0",
            "max-height": "$1"
        },
        ...
    },

   {
        "name": "Animation",
        "hasParams": true,
        "matcher": "Anim",
        "allowParamToValue": true,
        "styles": {
            "animation": "$0"
        }
   }
]
```